File: C#-part-2/03.Methods/09.Sorting array/SortingArray.cs
//Write a program that sorts a given array.
namespace _09.Sorting_array
{
    using System;
    using System.Linq;

    class SortingArray
    {
        class NumberAsArray
        {
            static void Main()
            {
                Console.ReadLine();
                var input = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
                Array.Sort(input);

                Console.WriteLine(string.Join(" ", input));
            }
        }
    }
}

File: JavaScript-Fundamentals/06.Functions/07.Sorting array/Sorting array.js
/** Problem 7. Sorting array
 * Write a method that returns the maximal element in a portion of array of integers starting at given index.
 * Using it write another method that sorts an array in ascending / descending order. Write a program that sorts a given array.
 * Input        Output
 6
 3 4 1 5 2 6    1 2 3 4 5 6
 */
function Order(args) {
    var arrayLength = args[0],
        array = args[1].split(' ').map(Number);

  // array.sort();
    return array;
}